content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Add a Primary Cluster
+description: Describes how to add a primary cluster to your mesh on a flat network.
+weight: 20
+keywords: [multicluster, primary-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add
```

Comment:
The opening paragraph should be more direct, explaining the task this document describes.  Maybe something like:

```
This guide expands your existing service mesh to include another a primary cluster.

Before you begin, you should have already installed your first primary cluster...
```

Reply:
The opening paragraph is consistent across all pages of the guide. The topic is not expanding the mesh but adding a primary cluster to it. The sentence is built to offer readers a ramp too.